python/test/unit/fem/test_assemble_cppimport.py

```diff
@@ -8,21 +8,20 @@
 import pathlib
 
 import cppimport
+import dolfinx.pkgconfig
```

Comment:
What happened to the import structure in this code?

Reply:
No idea!

Line is removed now.